jokes-cli.js

```diff
@@ -0,0 +1,147 @@
+#! /usr/bin/env node
+const jokes      = require('./jokes.js')
+const inquirer  = require('inquirer')
+
+/**
+ * Parse Arguments to get command and executing it
+ */
+let parseCommand = () => {
+    
+    /** remove two first arguments filename, path **/
+    var arguments = process.argv.slice(2)
+
+    /** If there is no argument print help **/
+    if (arguments.length == 0) {
+        printHelp()
+        process.exit()
+    }
+
+    /** Get Command from the first argument **/
+    switch (arguments[0]) {
+        case "add":
+            // TODO : add joke (after add firebase)
+            break
```

Comment:
I think leaving unused code has not added value, I'd drop this case. If we ever implement adding jokes, we'll add it